lib/softcover/builders/epub.rb

```diff
@@ -318,17 +317,43 @@ def html_with_math(chapter, images_dir, texmath_dir, pngs, options={})
           svg_filename = File.join(texmath_dir, "#{digest(output)}.svg")
           svg_filename_abspath = File.join("#{Dir.pwd}", svg_filename)
           File.write(svg_filename, output)
-          # Convert to PNG.
+          # Convert to PNG named:
           png_filename = svg_filename.sub('.svg', '.png')
           png_filename_abspath = svg_filename_abspath.sub('.svg', '.png')
           pngs << png_filename
+          #
+          # Settings for inline math in ePub / mobi
+          pt_scale_factor = 16  # scaling factor used for SVG --> PNG conversion
+          height_scale_factor = 0.50  # 1ex/1em for scaling math PNG's height
+          valign_scale_factor = 0.45  # 1ex/1em when scaling PNG's vertical-align
+          # these are used in three-step process below: Extract, Convert, Eeplace
+          # STEP1: Extract information from svg tag.
+          svg_height = svg['style'].scan(/height: (.*?);/).flatten.first
+          if svg_height
+            svg_height_in_ex = Float(svg_height.gsub('ex',''))
+            png_height = (svg_height_in_ex * height_scale_factor).to_s + 'em'
+           # MathJax sets SVG height in `ex` units but we wan `em` units for PNG
+          end
+          # Extract vertical-align css proprty for for inline math.
+          if svg.parent.parent.attr('class') == "inline_math"
+            vertical_align = svg['style'].scan(/vertical-align: (.*?);/).flatten.first
+            if vertical_align
+              valign_in_ex = Float(vertical_align.gsub('ex',''))
+              # png vertical-align in ems is the css equivalent of `depth` in TeX
+              png_valign = (valign_in_ex * valign_scale_factor).to_s + 'em'
+            else
+              png_valign = "0em"
```

Comment:
This sets a default `vertical-align:0em` for elements that did not have any (those where the svg rendered by mathjax didn't have a `vertical-align` property).